sregistry/main/google_build/delete.py

```diff
@@ -50,16 +51,22 @@ def delete(self, name):
 
     bot.debug("DELETE %s" % name)
 
+    # this is broken!!!
     for file_object in files:
```

Comment:
Is there a change in the storage API? Could you elaborate more on "this is broken?"

Reply:
Well I didn't test it, but there is no reference to variable `files` before it is used, which looks very suspicious to me...

Reply:
rofl, I don't even know how I messed that up. I'll take a look.